### PR TITLE
Update generator to take into account hashes generated by transforms

### PR DESCRIPTION
**Pull request by as6520**
This PR updates the generator to take into account the hashes that are being generated by the transforms. Depends on https://github.com/darpa-sail-on/videoaug/pull/5

### CODE REVIEW

File: gen_data/vid_utils.py [hunk 1]
```diff
@@ -72,6 +75,13 @@ def print_all_augs():
             augs_grouped[v.type].append(v.name + " --> " + str(k))
         print(json.dumps(augs_grouped, sort_keys=True, indent=4))
 
+    @staticmethod
+    def get_hash_for_augs():
+        hash_dict = {}
+        for k, v in AugWrapper.get_all_aug().items():
+            hash_dict[v.name] = hash(k)
```
Comment (Pikus16) [inline gen_data/vid_utils.py]: `hash` unfortunately will be random between different runs unless handled carefully (see https://docs.python.org/3/using/cmdline.html#envvar-PYTHONHASHSEED). Instead would recommend using hashlib here

Comment (as6520) [reply gen_data/vid_utils.py]: Hmm looked at hashlib quickly not sure how to use hashlib to determine hashes for python objects. Could you point me to an example of the same?

Comment (Pikus16) [reply gen_data/vid_utils.py]: Fixed here: https://github.com/darpa-sail-on/gen_data/pull/6/commits/47b81a16cff81bb10aaf4f645006cac78a8f044b

Comment (Pikus16) [reply gen_data/vid_utils.py]: Note: it doesn't use the augmenter itself, but just the string representation of the augmenter. Using the augmenter itself is a little more tricky.

**Comment by Pikus16** [comment]
Looks good to me, @as6520

File: gen_data/vid_utils.py (hunk 1 above)
Comment (as6520) [reply gen_data/vid_utils.py]: I see, LGTM

**Pull request merged**